dev/unused-images.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
nit: the script name is confusing, why not naming it like `clean-unused-images.sh` ?